conans/client/build/msbuild.py

```diff
@@ -11,6 +11,7 @@
 from conans.errors import ConanException
 from conans.model.conan_file import ConanFile
 from conans.model.version import Version
+from conans.tools import vcvars_command as tools_vcvars_command
```

Comment:
Didn't we say that conan does NOT import from ``conans.tools`` and that is exclusively for users/recipes? Besides 2 remaining things, that should be removed, that is the way it is now.

Reply:
@danimtb pointed out that **it is much more important not to expose an `output` argument in the public API/tool**. I agree with him, the signature for `MSBuild::get_version()` cannot require the user to provide an `output`.

Reply:
The problem is that MSBuild.get_version() prints that ugly warning, that users cannot fix.
Also the MSBuild binary_log feature, uses internally this functionality, printing the ugly warning message, and that cannot be fixed either by the user. 

This has to be fixed, outputting warnings that the user cannot fix doesn't make sense, no matter how "clean" our api is.

Reply:
The _patch_ by @danimtb avoids that warning (secondary goal) and do not expose the `output` argument (main goal).